lib/src/rules/use_named_constants.dart

```diff
@@ -60,15 +60,23 @@ class _Visitor extends SimpleAstVisitor<void> {
       final value = context.evaluateConstant(node).value;
```

Comment:
consider moving `value`, `library` and `nodeField` into the `if (element is ClassElement)` block?